### DISCUSSION TITLE
Búsqueda de imágenes

### DISCUSSION THREAD

**Discussion by AlanAcosta17** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#372] [Q&A]
Buen día compañeros. ¿Me podrían decir donde encontraron las imágenes para la práctica de css?

**Comment by IsmaelGomezPerez** [comment]
Espero que te sirvan.
[Imagen 1](https://www.freepik.com/free-photo/front-view-female-nurse-white-medical-suit-with-red-mask-injection-her-hands-blue_13935723.htm?query=nurses%20%20nursing#position=30&query=nurses++nursing)
[Imagen 2](https://www.freepik.com/free-photo/front-view-young-woman-sterile-mask-yellow-wall_12651631.htm#query=woman&position=0&from_view=author)
[Imagen3](https://www.freepik.com/free-photo/portrait-man-doctor-using-cellphone_7844038.htm#query=men%20doctor&position=7&from_view=author)
